ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest_SwiftUI.swift

```diff
@@ -17,4 +17,73 @@ class ActivityIndicatorTestSwiftUI: BaseTest {
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars.element(matching: NSPredicate(format: "identifier CONTAINS %@", controlName)).exists)
     }
+
+    // tests activity indicator's start/stop functionality
+    func testAnimating() throws {
+        app.switches["Hides when stopped"].tap()
+        XCTAssert(app.images.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is in progress")).exists)
+
+        app.switches["Animating"].tap()
+        XCTAssert(app.images.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is progress halted")).exists)
+    }
+
+    // ensures that activity indicator disappears when stopped
+    func testHidingWhenStoppedOn() throws {
+        XCTAssert(app.images.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is in progress")).exists)
+
+        app.switches["Animating"].tap()
+        XCTAssert(!app.images.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Activity Indicator that is in progress")).exists)
+    }
+
+    func testCustomColor() throws {
+        XCTAssert(app.images.containing(NSPredicate(format: "identifier MATCHES %@", "Activity Indicator.*of default color.*")).element.exists)
+
+        app.switches["Uses custom color"].tap()
+        let RGBAValues = "[0.0, 0.47058823529411764, 0.8313725490196079, 1.0]"
```

Comment:
Just for my understanding -- I'm guessing that these RGBA values are that of the ones in the app? Those are some crazy numbers haha

Reply:
Yep, it's for the custom blue.